pyproject.toml

```diff
@@ -47,7 +47,7 @@ test = [
 runtime = [
     "accelerate", # needed by low_cpu_mem_usage
     "huggingface_hub",
-    "diffusers[torch]>=0.27.0",
+    "diffusers>=0.27.0",
```

Comment:
why?

Reply:
Considering the necessity of including both "diffusers[torch]>=0.27.0" and "torch" in the dependencies.

Reply:
diffusers[torch]>=0.27.0 will try to install torch==2.4.1